layer/clients/logged_data_service.py

```diff
@@ -70,14 +75,19 @@ def get_logged_data(
         )
 
     def log_model_metric(
-        self, train_id: UUID, tag: str, points: List[ModelMetricPoint]
+        self,
+        train_id: UUID,
+        tag: str,
+        points: List[ModelMetricPoint],
+        metric_group_uuid: UUID,
```

Comment:
nit: `metric_group_id`, to be consistent with `train_id` above and also `LoggedMetricGroupId`